lua/ulx/modules/sh/cfc_curse.lua

```diff
@@ -129,9 +220,17 @@ local function silentCursePlayers( callingPlayer, targetPlayers, effectName, dur
 end
 
 
+local curseOptions = table.Copy( CFCUlxCurse.GetEffectNames() )
+table.insert( curseOptions, "random" )
+
+for i = 1, 10 do
+    table.insert( curseOptions, tostring( i ) )
+end
```

Comment:
Add `random` to the auto-fill list, along with 1 thru 10. Higher numbers can be provided, but these are just to indicate that numbers can now be provided.